src/components/charts/tooltip-chart/tooltip-chart.js

```diff
@@ -118,7 +118,7 @@ class TooltipChart extends PureComponent {
             content.payload.length > 0 &&
             this.sortByValue(content.payload).map(y => {
               const hasDataKey = !!y.dataKey;
-              const labelName = y.dataKey || y.name;
```

Comment:
After recharts update `y.dataKey="value"` (see pie-chart.js file `<Pie ... dataKey="value" />`) but before it was  `dataKey=undefined`. I'm not sure where is the difference, but to make this work again I switched the places of variables here. We always get  `y.dataKey==y.name` or `y.dataKey==undefined` so it should work without breaking other charts